src/Orleans.Transactions/Hosting/SiloBuilderExtensions.cs

```diff
@@ -13,25 +13,24 @@ public static class SiloBuilderExtensions
         /// <summary>
         /// Configure cluster to use the distributed TM algorithm
         /// </summary>
-        public static ISiloHostBuilder UseDistributedTM(this ISiloHostBuilder builder)
+        public static ISiloHostBuilder UseDistributedTM(this ISiloHostBuilder builder, bool withReporter = true)
```

Comment:
This new parameter is not obvious and hence could benefit from a doc comment.

Slightly prefer naming it `reportStatistics`.

I'm not entirely convinced that this needs to be exposed - or could we implement this using the options pattern somehow?

Reply:
Issue here is that users can provide their own statistics reporting.  So they can configure tm with reporting, or without, and provide their own.  I'll add a comment.
As for being part of the options, the Tm has no options now and this is really about what we're setting up in the container, not how the TM is configured. :/